packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -257,7 +259,10 @@ Uploading...
   }
 
   async deploy() {
```

Comment:
this method is too long. I suggest splitting in 
deploy() {
try {
 buildIfChanged... (no need to wrap in new method)
  new zip() method
 new deployBundle() method
new method displayProviders()
} finally () {
remove tmp and bundle
}

Reply:
changed to:
deploy(){
try{
buildIfChanged <- same logic we previously had here
createBundle() <- this is the responsible of generating the bundle
deployBundle() <- all logic of posting the data goes here
displayDeployResults() <- we already had a displayProviders method, so I have encapsulated it into displayDeployResults method
} finally{
removeSync both tmp and bundle
}
}

Reply:
thanks for your feeddback Dani! 👍